lib/games/single_game.dart

```diff
@@ -1107,3 +1127,24 @@ class _SingleGameState extends State<SingleGame> with TickerProviderStateMixin {
     return null;
   }
 }
+
+class ShowHelpDrawer extends StatelessWidget {
```

Comment:
Why is this in the same file single_game.dart? It should ideally be in its own file